includes/admin/class-wc-rest-payments-settings-controller.php

```diff
@@ -75,32 +77,60 @@ public function register_routes() {
 						],
 						'validate_callback' => 'rest_validate_request_arg',
 					],
-					'is_manual_capture_enabled'         => [
+					'is_manual_capture_enabled'          => [
 						'description'       => __( 'If WooCommerce Payments manual capture of charges should be enabled.', 'woocommerce-payments' ),
 						'type'              => 'boolean',
 						'validate_callback' => 'rest_validate_request_arg',
 					],
-					'is_test_mode_enabled'              => [
+					'is_test_mode_enabled'               => [
 						'description'       => __( 'WooCommerce Payments test mode setting.', 'woocommerce-payments' ),
 						'type'              => 'boolean',
 						'validate_callback' => 'rest_validate_request_arg',
 					],
-					'account_statement_descriptor'      => [
+					'account_statement_descriptor'       => [
 						'description'       => __( 'WooCommerce Payments bank account descriptor to be displayed in customers\' bank accounts.', 'woocommerce-payments' ),
 						'type'              => 'string',
 						'validate_callback' => 'rest_validate_request_arg',
 					],
-					'is_digital_wallets_enabled'        => [
+					'is_digital_wallets_enabled'         => [
 						'description'       => __( 'If WooCommerce Payments express checkouts should be enabled.', 'woocommerce-payments' ),
 						'type'              => 'boolean',
 						'validate_callback' => 'rest_validate_request_arg',
 					],
-					'digital_wallets_enabled_locations' => [
+					'digital_wallets_enabled_locations'  => [
 						'description'       => __( 'Express checkout locations that should be enabled.', 'woocommerce-payments' ),
 						'type'              => 'array',
 						'items'             => [
 							'type' => 'string',
-							'enum' => Digital_Wallets_Locations::toArray(),
+							'enum' => array_keys( $wcpay_form_fields['payment_request_button_locations']['options'] ),
+						],
+						'validate_callback' => 'rest_validate_request_arg',
+					],
+					'digital_wallets_button_action_type' => [
+						'description'       => __( '1-click checkout button types.', 'woocommerce-payments' ),
+						'type'              => 'string',
+						'items'             => [
+							'type' => 'string',
+							'enum' => array_keys( $wcpay_form_fields['payment_request_button_type']['options'] ),
+						],
+						'validate_callback' => 'rest_validate_request_arg',
+					],
+					'digital_wallets_button_size'        => [
+						'description'       => __( '1-click checkout button sizes.', 'woocommerce-payments' ),
+						'type'              => 'string',
+						'items'             => [
+							'type' => 'string',
+							// it can happen that `$wcpay_form_fields['payment_request_button_size']` is empty (in tests) - fixing temporarily.
```

Comment:
Will this be addressed by another PR soon or is it something to be addressed in the future? This should probably be fixed on the tests' side.

Reply:
I agree that it should be resolved in the test' side. I timeboxed it, but I haven't been able to find a solution, so I just added the `isset` check so that I could move forward.
It is happening on unrelated tests, too - and it's due to the fact that `payment_request_button_size` is not set when the "grouped settings" flag is `false`.

I will remove the check after we deploy the feature with the feature flag enabled 👍